Sources/Danger/Extensions/DateFormatterExtensions.swift

```diff
@@ -3,10 +3,44 @@ import Foundation
 extension DateFormatter {
     // A default implementation of an iso8601 DateFormatter
     public static var defaultDateFormatter: DateFormatter {
-        let dateFormatter = DateFormatter()
-        dateFormatter.locale = Locale(identifier: "en_US_POSIX")
-        dateFormatter.timeZone = TimeZone(secondsFromGMT: 0)
-        dateFormatter.dateFormat = "yyyy-MM-dd'T'HH:mm:ssZZZZ"
+        let dateFormatter = OptionalFractionalSecondsDateFormatter()
         return dateFormatter
     }
 }
+
+private final class OptionalFractionalSecondsDateFormatter: DateFormatter {
```

Comment:
GitLab returns the dates with milliseconds in it.
Looks that `decoder.dateDecodingStrategy = .iso8601` doesn't support that, then I created a new dateFormatter that can be used in `decoder.dateDecodingStrategy = .formatted`